pkg/push/domain/push.go

```diff
@@ -157,6 +165,33 @@ func (p *Push) ExistTag(findTag string) bool {
 	return false
 }
 
+func (p *Push) AddTag(tag string) error {
+	if contains(tag, p.Tags) {
+		return nil
+	}
+	p.Tags = append(p.Tags, tag)
+	return nil
+}
+
+func (p *Push) RemoveTag(tag string) error {
```

Comment:
To avoid compatibility issues with the current add and delete tag functions, I have added these new ones.
I will delete the old functions being used by the current console when the new one is released.